feg/cloud/go/services/feg_relay/gw_to_feg_relay/servicers/session_proxy_relay.go

```diff
@@ -71,33 +72,56 @@ func (s *RelayRouter) UpdateSession(
 			req.UsageMonitors = append(req.UsageMonitors, m)
 		}
 	}
-	resultChan := make(chan *protos.UpdateSessionResponse, len(reqMap))
+
+	/*Each request in reqMap is segregated into two requests and relays each request to respective proxies (session_proxy and n7n40_proxy).
+	Hence length of resultChan is len(reqMap) * 2
+	*/
```

Comment:
please format your comments similar to the rest of codebase

Reply:
Done.